docs/source/tutorials/cascaded_architecture.rst

```diff
@@ -0,0 +1,154 @@
+
+     "contrast_params": {
+         "training_validation": ["T2w"],
+         "testing": ["T2w"],
+         "balance": {}
+     }
+
+- ``transformation:CenterCrop:size``: Crop size in pixel. Images will be cropped or padded to fit these dimensions. This
+  allows all the images to have the same size during training. Since the images will be cropped around the spinal cord,
+  the image size can be reduced. The ``preprocessing`` parameter indicates this transformation will only be applied
+  once at the beginning of the training.
+
+  .. code-block:: xml
+
+     "CenterCrop": {
+         "size": [64, 64],
+         "preprocessing": true
+     }
+
+Train model
+-----------
+
+Once the configuration file is ready, run the training. `ivadomed` has an option to find optimal threshold to binarize
+the test data to find the best balance between false positives and false negatives. Add the flag `-t` with an increment
+between 0 and 1 to get best threshold on the validation set (e.i. -t 0.1 will return the best threshold between 0.1,
+0.2, ..., 0.9)
+
+.. code-block:: bash
+
+   ivadomed -c config.json -t 0.01
+
+.. note::
+
+   If a `compatible GPU <https://pytorch.org/get-started/locally/>`_ is available, it will be used by default. Otherwise, training will use the CPU, which will take
+   a prohibitively long computational time (several hours).
+
+At the end of the training, the optimal threshold will be indicated:
+
+.. code-block:: console
+
+   Running threshold analysis to find optimal threshold
+	Optimal threshold: 0.01
+	Saving plot: spineGeneric/roc.png
+
+
+Visualize training data
+-----------------------
+
+Tensorboard helps visualize the augmented input images, the model's prediction, the groud truth, the learning curves, and
+more. To access this data during or after training, use the following command-line:
+
+.. code-block:: bash
+
+   tensorboard --logdir <PATH_TO_LOG_DIRECTORY>
+
+The following should be displayed in the terminal:
+
+.. code-block:: console
+
+   Serving TensorBoard on localhost; to expose to the network, use a proxy or pass --bind_all
+   TensorBoard 2.2.1 at http://localhost:6006/ (Press CTRL+C to quit)
+
+Open your browser and type the URL provided, in this case ``http://localhost:6006/``.
+In the scalars folder, the evolution of metrics, learning rate and loss through the epochs can be visualized.
+
+.. image:: ../../../images/tensorboard_scalar.png
+   :align: center
+
+In the image folder, the training and validation ground truth, input images and predictions are displayed. With this
+feature, it is possible to visualize the cropping from the first model and confirm that the spinal cord
+was correctly located.
+
+.. image:: ../../../images/tensorboard_images.png
+   :align: center
+
+Evaluate model
+--------------
+- ``testing_parameters:binarize_prediction``: Threshold at which predictions are binarized. Before testing the model,
+  modify the binarization threshold to have a threshold adapted to the data:
```

Comment:
I don't think we need to do it manually: see [here](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/main.py#L199).

Reply:
See previous comment, but we could add the modification of the working config file (currently only the config files in the log directory are modified)